Gemfile

```diff
@@ -13,8 +13,9 @@ group :development, :test do
   gem 'climate_control'
 end
 
-gem 'azure_mgmt_resources', '~>0.5.0'
-gem 'azure_mgmt_compute',   '~>0.5.0'
-gem 'azure_mgmt_network',   '~>0.5.0'
-gem 'azure_mgmt_storage',   '~>0.5.0'
-gem 'haikunator',           '~>1.1'
\ No newline at end of file
+gem 'azure_mgmt_resources', '~>0.6.0'
+gem 'azure_mgmt_compute',   '~>0.6.0'
+gem 'azure_mgmt_network',   '~>0.6.0'
+gem 'azure_mgmt_storage',   '~>0.6.0'
+gem 'haikunator',           '~>1.1'
+
```

Comment:
That's strange. I tried using checking out your branch and seeing this file in RubyMine. I can see two lines. Let me know if you are using RubyMine and it doesn't show two lines then it must be some of my local configurations.